running_2000.py

```diff
@@ -0,0 +1,12 @@
+import time
+
+
+def timer(f, *structure, **structure_keyword):
+    start_time = time.time()
+    map(f, filter(None, [structure, structure_keyword]))
```

Comment:
Why do we need the map and the filter here?

Reply:
I'm using the filter to find which of the structure is passed to the function and just then using the map to apply the needed function. If I won't use the fillter I will cause an exception if ill to try to activate the function on null structure.